get_if_addrs-sys/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
```

Comment:
Make sure we call the crates "get_if_addrs" / "get_if_addrs-sys", not "ifaddrs" / "ifaddrs-sys". Check these fields:
- description
- documentation
- name
- repository